starknet_py/net/client.py

```diff
@@ -26,6 +27,11 @@
     chain_from_network,
 )
 from starknet_py.net.networks import Network, net_address_from_net
+from starknet_py.transaction_exceptions import (
+    TransactionFailedException,
```

Comment:
It is more conventional to end custom error classes with `Error` instead of `Exception`

https://namingconvention.org/python/exception-naming.html